### PR TITLE
Minor edits to therapeutics & add a summary paragraph

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

These are some superficial edits to the therapeutics section. I also added a short summary paragraph at the bottom and updated the content to reflect that remdesivir received FDA approval yesterday.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -72,26 +72,24 @@ At the outset of the COVID-19 pandemic, it did not have any have any FDA-approve
...
+It has also been found to inhibit SARS-CoV-2, showing synergy with chloroquine _in vitro_ [@doi:10.1038/s41422-020-0282-0].
 The effectiveness of remdesivir for treating patients with COVID-19 is currently under investigation.
+
 Remdesivir was first used on some COVID-19 patients under compassionate use guidelines [@doi:10.1126/science.abb7243, @doi:10.1056/NEJMoa2001191; @doi:10.1101/2020.03.09.20032896].
 All were in late stages of COVID-19 infection, and these reports were inconclusive about the drug's efficacy.
 Gilead Sciences, the maker of remdesivir, led a recent publication that reported outcomes for compassionate use of the drug in 61 patients hospitalized with confirmed COVID-19.
-Here, 200mg of remdesivir was administered intravenously on day 1, followed by a further 100mg/day for 9 days [@doi:10.1056/NEJMoa2007016].
-There were significant issues with the study design or lack thereof.
+Here, 20 0mg of remdesivir was administered intravenously on day 1, followed by a further 100 mg/day for 9 days [@doi:10.1056/NEJMoa2007016].
```
Comment (RLordan) [inline content/20.treatments.md:83]: 20 0mg?

File: content/20.treatments.md [hunk 2]
```diff
@@ -125,17 +124,17 @@ Several FDA-approved drugs target proteases, including lopinavir and ritonavir f
...
-Investigation of possible PIs that work against SARS-CoV-2 has been driven by computational predictions, leading to the computer-aided design of a Michael acceptor inhibitor, N3, to target a protease critical to SARS-CoV-2 replication.
+Investigation of possible PIs that would work against SARS-CoV-2 has been driven by computational predictions, leading to the computer-aided design of a Michael acceptor inhibitor, N3, to target a protease critical to SARS-CoV-2 replication.
 
 Discovery of the N3 mechanism arose from interest in the two polyproteins encoded by the SARS-CoV-2 replicase gene, pp1a and pp1ab, that are critical for viral replication and transcription [@doi:10.1038/s41586-020-2223-y].
 These polyproteins must undergo proteolytic processing. 
-This processing is usually conducted by Mpro, a 33.8-kilodaltons (kDa) SARS-CoV-2 protease that is therefore fundamental to viral replication and transcription.
+This processing is usually conducted by Mpro, a 33.8-kilodalton (kDa) SARS-CoV-2 protease that is therefore fundamental to viral replication and transcription.
```
Comment (RLordan) [inline content/20.treatments.md:131]: plural kilodaltons is fine I believe, not important though

File: content/20.treatments.md [hunk 3]
```diff
@@ -167,16 +166,16 @@ In the United States, for example, they are prescribed well over 100,000,000 tim
...
 These studies of randomized intervention will provide important data for understanding whether exposure to ACEis or ARBs is associated with COVID-19 outcomes.
 Additional information about ACE2, observational studies of ACE inhibitors and ARBs in COVID-19, and clinical trials on this topic have been summarized [@url:http://www.nephjc.com/news/covidace2].
 
 #### Broad-Spectrum Pharmaceuticals
 
-Due to the urgent nature of the COVID-19 pandemic, many of the pharmaceutical agents that have been widely publicized as having possible therapeutic or prophylactic effects are broad-spectrum pharmaceuticals.
-These treatments, which in most cases pre-date the COVID-19 pandemic, are not specifically targeted at the virus itself or at the host receptors it relies on, but rather induce broad shifts in host biology that are hypothesized as potential inhibitors of the virus.
+Due to the urgent nature of the COVID-19 pandemic, many of the pharmaceutical agents that have been widely publicized as having possible therapeutic or prophylactic effects are broad-spectrum pharmaceuticals that pre-date the COVID-19 pandemic..
```
Comment (RLordan) [inline content/20.treatments.md:177]:
```suggestion
Due to the urgent nature of the COVID-19 pandemic, many of the pharmaceutical agents that have been widely publicized as having possible therapeutic or prophylactic effects are broad-spectrum pharmaceuticals that pre-date the COVID-19 pandemic.
```

**Review by RLordan — APPROVED** [review]
Looks great!

File: content/20.treatments.md (hunk 1 above)
Comment (rando2) [reply content/20.treatments.md:83]:
```suggestion
Here, 200 mg of remdesivir was administered intravenously on day 1, followed by a further 100 mg/day for 9 days [@doi:10.1056/NEJMoa2007016].
```

**Review by rando2 — COMMENTED** [review]
Thank you @RLordan! Glad to not be inventing my own unit of weight.

**Pull request merged**